test_finbert_python.py

```
@@ -0,0 +1,60 @@
+    results.append({
+        "text": text,
+        "sentiment": sentiment,
+        "confidence": confidence,
+        "label": result["label"]
+    })
+
+# Save results
+with open("finbert_test_results.json", "w") as f:
+    json.dump(results, indent=2, fp=f)
+
+print(f"\n✅ Results saved to finbert_test_results.json")
```

Ruff (0.14.1)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)